public/css/signup.css

```diff
@@ -0,0 +1,74 @@
+
+h1{
+    text-align: center;
+    font-family: Roboto;
+
+}
+
+.msg{
+    text-align: center;
+    font-family: Roboto;
+    color: red;
+    
+}
+.containerr , .submitdiv{
+    /* display: grid;
+    grid-template-columns: repeat( auto-fill, minmax(400px, 1fr) );
+    grid-gap:10px; */
+    margin:0 auto;
+    width:70%;
+}
+.c{
```

Comment:
`.c` Really?
you should choose proper names for your classes and check the spelling also, if you are not sure, ask `google`